### DISCUSSION TITLE
UnsupportedClassVersionError when running CLI v1.30.0

### DISCUSSION THREAD

**Discussion by sschutten** [structurizr/cli#117] [Q&A] [answered]
### Description

Since version v1.30.0 I get the following exception when I try to run the Structurizr CLI:

```
Exception in thread "main" java.lang.UnsupportedClassVersionError: com/structurizr/cli/StructurizrCliApplication has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

I'm running the Java 8 Update 371 runtime environment (the latest as of writing). The [installation page](https://github.com/structurizr/cli/blob/master/docs/local-installation.md) mentions I will need Java version 11+, but I can't find a runtime environment newer than 8. Do I really need to install the JDK or is there a more recent version of the runtime I'm unaware of?

### Steps to reproduce

1. Install Java 8 Update 371
2. Download Structurizr CLI 1.30.0
3. Run structurizr

### Configuration

Structurizr CLI v1.30.0
Java 8 Update 371
Windows 11

### Severity

Minor

### Priority

Low

### Resolution

I have no budget, please fix this for free

**Comment by simonbrowndotje** [comment, marked as answer]
Active support for Java 8 ended in 2022, so you will need to upgrade your JDK/JRE to 11+.
